exec/src/main/java/com/optum/sourcehawk/exec/FlattenCommand.java

```diff
@@ -0,0 +1,74 @@
+package com.optum.sourcehawk.exec;
+
+import com.optum.sourcehawk.core.scan.FlattenResult;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import picocli.CommandLine;
+
+/**
+ * CLI entry point for executing Sourcehawk flatten command
+ *
+ * @author Christian Oestreich
+ */
+@Slf4j
+@CommandLine.Command(
+        name = FlattenCommand.COMMAND_NAME,
+        aliases = {"flat", "merge", "converge"},
+        description = "Flatten the sourcehawk.yml onto system by recursively reading and merging local and remote configurations"
+)
+class FlattenCommand extends AbstractCommand {
```

Comment:
Can we make this a little bit more specific, `FlattenConfig`, similar to `ValidateConfig`, just to be clear on what its flattening.

Reply:
I don't think we need all other options from `AbstractCommand` here...  Maybe similar to `ValidateConfigCommand` you just take the config file path?

Reply:
https://picocli.info/#mixin-annotation

Reply:
Got rid of a bunch of code